src/main/java/com/example/SpotifyApi/SpotifyApiApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.example.SpotifyApi;
```

Comment:
why com.example.SpotifyApi? we are using ua.lviv.iot as domain (or ua.lpnu.csa )